tools/dump_tool/HowToUseDumpTool.md

```diff
@@ -207,8 +268,8 @@ Call Trace of Crashed Task :[appmain] with pid :2 and state :TSTATE_TASK_RUNNING
 [<40c9fec>] task_start+0x64         [Line 173 of \"task/task_start.c\"]
 ********************************************************************
 ```
-## How to port RAMDUMP
-To port ramdump for a new board, do the following steps:
+## How to port DUMP Tool
```

Comment:
Let's go with "How to port RAM and Userfs Functionality"?

Reply:
Hm.. Do we need to mention `RAM and Userfs` here?
Because there is no difference between RAM and Userfs and no other requirement.
How about `How to port memory dump functionality`?

Reply:
Done